tox.ini

```diff
@@ -1,39 +1,21 @@
 [tox]
-envlist = py38-django{22,30}
+envlist = py38-django{22,30,31,32}
 
 [doc8]
-; D001 = Line too long
-ignore=D001
+ignore = D001
 
 [pycodestyle]
 exclude = .git,.tox,migrations
 max-line-length = 120
 
 [pydocstyle]
-; D101 = Missing docstring in public class
-; D200 = One-line docstring should fit on one line with quotes
-; D203 = 1 blank line required before class docstring
-; D212 = Multi-line docstring summary should start at the first line
-; D215 = Section underline is over-indented (numpy style)
-; D404 = First word of the docstring should not be This (numpy style)
-; D405 = Section name should be properly capitalized (numpy style)
-; D406 = Section name should end with a newline (numpy style)
-; D407 = Missing dashed underline after section (numpy style)
-; D408 = Section underline should be in the line following the section’s name (numpy style)
-; D409 = Section underline should match the length of its name (numpy style)
-; D410 = Missing blank line after section (numpy style)
-; D411 = Missing blank line before section (numpy style)
-; D412 = No blank lines allowed between a section header and its content (numpy style)
-; D413 = Missing blank line after last section (numpy style)
-; D414 = Section has no content (numpy style)
```

Comment:
why these lines are removed?

Reply:
These were removed through one of the Codemod scripts(https://github.com/edx/repo-tools/blob/master/edx_repo_tools/codemods/django3/tox_modernizer.py) that we run as part of edX's standards. If we need to ignore these warnings moving on we can add them back.